trident/absorption_spectrum/absorption_line.py

```diff
@@ -222,6 +225,83 @@ def tau_profile(lambda_0, f_value, gamma, v_doppler, column_density,
 
     return (lambda_bins, tauphi)
 
+def tau_profile_21cm(lambda_0, f_value, gamma, temperature, number_density,
+                h_now, delta_v=None, delta_lambda=None,
+                lambda_bins=None, n_lambda=12000, dlambda=0.01):
+    r"""
+    Create an optical depth vs. wavelength profile for the
+    21 cm forest. The optical depth is calculated using eq. 1 in https://arxiv.org/abs/1510.02296. At this point in the implementation, we make a very reasonable assumption that (1+ 1/H(z) dv/dr) ~ 1. 
```

Comment:
Can you break this into multiple lines? That will make it easier to read in the reference documentation.

Reply:
Updated .